ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -39,12 +42,24 @@ private[kvutils] trait CommitContext {
 
   /** Set a value in the output state. */
   def set(key: DamlStateKey, value: DamlStateValue): Unit = {
-    if (!outputs.contains(key)) {
-      outputOrder += key
+    if (inputAlreadyContains(key, value)) {
+      logger.error(s"Identical output found for key $key")
```

Comment:
Why an error?

Reply:
Will change that before merging -- wanted to see if we actually hit this.

Reply:
Changed it back to `TRACE`.